packages/tenant-process/src/services/tenantService.ts

```diff
@@ -652,8 +652,8 @@ export function tenantServiceBuilder(
         authData,
         correlationId,
         logger,
-      }: WithLogger<AppContext<UIAuthData | M2MAuthData>>
-    ): Promise<void> {
+      }: WithLogger<AppContext<UIAuthData | M2MAuthData | M2MAdminAuthData>>
+    ): Promise<WithMetadata<Tenant>> {
```

Comment:
As done for attributes/delegations, I would update this test to test the entire result object including metadata

See https://github.com/pagopa/interop-be-monorepo/pull/1849/files#diff-2ab69d62d3cac341a38dac792805f097aee52ca413a8806d8b976398bf47d1afR82